src/behaviors.rs

```diff
@@ -13,10 +13,9 @@ pub enum RemoveBehavior {
     /// Use this behavior with caution.
     ///
     /// ```
-    /// use id_tree::Tree;
-    /// use id_tree::Node;
-    /// use id_tree::RemoveBehavior;
-    ///
+    /// # use id_tree::Tree;
+    /// # use id_tree::Node;
+    /// # use id_tree::RemoveBehavior;
     /// let mut tree: Tree<i32> = Tree::new();
```

Comment:
Yeah,  that's a good point.  And I think it's probably fine to just use `*` in the examples.  I'll go with that solution for now.  Thanks!